docs/en/sql-reference/sql-statements/Administration/CREATE_STORAGE_VOLUME.md

```diff
@@ -43,7 +43,7 @@ The table below lists all available properties of storage volumes. Following the
 | ----------------------------------- | ------------------------------------------------------------ |
 | enabled                             | Whether to enable this storage volume. Default: `false`. Disabled storage volume cannot be referenced. |
 | aws.s3.region                       | The region in which your S3 bucket resides, for example, `us-west-2`. |
-| aws.s3.endpoint                     | The endpoint URL used to access your S3 bucket, for example, `https://s3.us-west-2.amazonaws.com`. |
+| aws.s3.endpoint                     | The endpoint URL used to access your S3 bucket, for example, `https://s3.us-west-2.amazonaws.com`. From v3.3.0 onwards, Amazon S3 Express One Zone Storage is supported. |
```

Comment:
From v3.3.0 onwards, the Amazon S3 Express One Zone storage class is supported, for example, `https://s3express.us-west-2.amazonaws.com`.